netmiko/nokia/nokia_sros_ssh.py

```diff
@@ -248,7 +248,7 @@ def check_file_exists(self, remote_cmd=""):
 
         if self.direction == "put":
             if not remote_cmd:
-                remote_cmd = self._file_cmd_prefix() + "file dir {}/{}".format(
+                remote_cmd = self._file_cmd_prefix() + "file dir {}\\{}".format(
```

Comment:
I wonder why we needed to use `\\` here? Why not keeping `/`?

Reply:
Agree better to keep single \ like in other lines.